js/package.json

```diff
@@ -39,7 +39,7 @@
     "@angular/material": "10.2.2",
     "@jupyter-widgets/base": "^1.1 || ^2 || ^3 || ^4",
     "bootstrap": "3.3.7",
-    "chart.js": "3.2.0",
+    "chart.js": "3.7.0",
```

Comment:
I am pretty sure the version is then conflict with other two chart  library below. Can you check this file and make sure bandsplat js library can be installed by npm or yarn?

Reply:
It seems working fine with v3.7.0. I did see any problem.

Reply:
Thanks for checking, but if I go to the js folder I can not use yarn or npm to install. There is deps conflicting. My suggestion is we can fix that in another PR, would you mind to open an issue for that and investigate a bit on the annotation package? For this PR, I think it's all good then.